pages/common/lame.md

```diff
@@ -0,0 +1,17 @@
+# lame
+
+> LAME is a high quality MPEG Audio Layer III (MP3) encoder licensed under the LGPL.
```

Comment:
```suggestion
> Encode mp3 files.
```
Use imperative mood in the description https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#imperative-mood